debug/requirements.txt

```diff
@@ -0,0 +1 @@
+pexpect==4.2.1
```

Comment:
I just picked the latest version because I didn't know exactly which version was required. I've changed the requirement to >=4.0.0 in https://github.com/riscv/riscv-tests/pull/24/commits/de9053b96294acbcf2c2de46634720881f09f3a3, which I assume should be fairly safe.